include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -440,19 +441,22 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 			const core::vectorSIMDf fOutExtent(outExtentLayerCount);
 			const auto fScale = fInExtent.preciseDivision(fOutExtent);
 			const auto halfTexelOffset = fScale*0.5f-core::vectorSIMDf(0.f,0.f,0.f,0.5f);
-			const auto startCoord =  [&halfTexelOffset,&scaledKernelX,&scaledKernelY,&scaledKernelZ]() -> core::vectorSIMDi32
+			const auto startCoord =  [&halfTexelOffset,state]() -> core::vectorSIMDi32
```

Comment:
why are you passing `state` by value instead of by reference?

Reply:
Because `state` is just a raw pointer (and we don't mutate the pointer itself, inside the lambda)
https://github.com/Devsh-Graphics-Programming/Nabla/blob/75a7a200fadfb04397b86c631f0d3a085e10b86e/include/nbl/asset/filters/CBlitImageFilter.h#L292

Reply:
ah ok, wasn't apparent without Intellisense XD